crates/holochain/Cargo.toml

```diff
@@ -193,9 +191,8 @@ path = "src/bin/holochain/main.rs"
 workspace = true
 
 [features]
-default = ["sqlite-encrypted", "tx2", "tx5", "metrics_influxive", "wasmer_sys"]
+default = ["chc", "sqlite-encrypted", "tx5", "metrics_influxive", "wasmer_sys"]
```

Comment:
why is chc becoming a default feature as part of this PR?

Reply:
Probably a bad merge, after fixing the same merge conflicts dozens of times I was bound to get it wrong at some point :)